py3status/modules/file_status.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Display if a file or dir exists.
+
+Configuration parameters:
```

Comment:
same as the other process module, please state the defaults (and path shouldn't have one imo)